{"uid":"a729c66727d58961","name":"Тест доступности Подуслуг по Услуге","fullName":"tests.test_service#test_service_pages","historyId":"4640de6902f221b70caa7f772a048086","time":{"start":1733370188841,"stop":1733370188841,"duration":0},"description":"Этот тест проверяет доступность страниц услуг","descriptionHtml":"<p>Этот тест проверяет доступность страниц услуг</p>\n","status":"broken","statusMessage":"aiohttp.client_exceptions.ClientConnectorError: Cannot connect to host gosuslugi29.ru:443 ssl:False [Connection reset by peer]","statusTrace":"self = <asyncio.sslproto.SSLProtocol object at 0x7fb32b5fbf50>\n\n    def _do_handshake(self):\n        try:\n>           self._sslobj.do_handshake()\n\n/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/asyncio/sslproto.py:560: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <ssl.SSLObject object at 0x7fb32bb60510>\n\n    def do_handshake(self):\n        \"\"\"Start the SSL/TLS handshake.\"\"\"\n>       self._sslobj.do_handshake()\nE       ssl.SSLWantReadError: The operation did not complete (read) (_ssl.c:1006)\n\n/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/ssl.py:979: SSLWantReadError\n\nDuring handling of the above exception, another exception occurred:\n\nself = <aiohttp.connector.TCPConnector object at 0x7fb32bd63250>\nreq = <aiohttp.client_reqrep.ClientRequest object at 0x7fb33035b9d0>\ntimeout = ClientTimeout(total=300, connect=None, sock_read=None, sock_connect=None)\nclient_error = <class 'aiohttp.client_exceptions.ClientConnectorError'>\nargs = (functools.partial(<class 'aiohttp.client_proto.ResponseHandler'>, loop=<_UnixSelectorEventLoop running=False closed=False debug=False>), '77.83.151.43', 443)\nkwargs = {'family': <AddressFamily.AF_INET: 2>, 'flags': <AddressInfo.AI_NUMERICHOST|AI_NUMERICSERV: 1028>, 'local_addr': None, 'proto': 6, ...}\n\n    async def _wrap_create_connection(\n        self,\n        *args: Any,\n        req: \"ClientRequest\",\n        timeout: \"ClientTimeout\",\n        client_error: Type[Exception] = ClientConnectorError,\n        **kwargs: Any,\n    ) -> Tuple[asyncio.Transport, ResponseHandler]:\n        try:\n            async with ceil_timeout(timeout.sock_connect):\n>               return await self._loop.create_connection(*args, **kwargs)  # type: ignore[return-value]  # noqa\n\n/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/aiohttp/connector.py:980: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/asyncio/base_events.py:1113: in create_connection\n    transport, protocol = await self._create_connection_transport(\n/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/asyncio/base_events.py:1146: in _create_connection_transport\n    await waiter\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <_SelectorSocketTransport closed fd=43>\ndata = b\"\\x16\\x03\\x01\\x02\\x00\\x01\\x00\\x01\\xfc\\x03\\x03\\r\\x8d\\xc0@\\xaf\\x7f\\x1c\\x12\\xec3@\\x8dE\\x01jC\\x8di\\x92\\x95,\\x99\\x92\\xe4\\x...00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\"\n\n    def write(self, data):\n        if not isinstance(data, (bytes, bytearray, memoryview)):\n            raise TypeError(f'data argument must be a bytes-like object, '\n                            f'not {type(data).__name__!r}')\n        if self._eof:\n            raise RuntimeError('Cannot call write() after write_eof()')\n        if self._empty_waiter is not None:\n            raise RuntimeError('unable to write; sendfile is in progress')\n        if not data:\n            return\n    \n        if self._conn_lost:\n            if self._conn_lost >= constants.LOG_THRESHOLD_FOR_CONNLOST_WRITES:\n                logger.warning('socket.send() raised exception.')\n            self._conn_lost += 1\n            return\n    \n        if not self._buffer:\n            # Optimization: try to send now.\n            try:\n>               n = self._sock.send(data)\nE               ConnectionResetError: [Errno 104] Connection reset by peer\n\n/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/asyncio/selector_events.py:1061: ConnectionResetError\n\nThe above exception was the direct cause of the following exception:\n\nrequest = <SubRequest 'service_pages' for <Function test_service_pages>>\n\n    @pytest.fixture(scope=\"session\")\n    async def service_pages(request):\n        pages = []\n        async with aiohttp.ClientSession() as session:\n>           pages = await get_service_pages(session, request.config.services)\n\ntests/test_service.py:62: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests/test_service.py:33: in get_service_pages\n    await responses\nutils/utils.py:146: in fetch_url\n    async with session.request(**req, url=buildurl(**UrlPart)) as response:\n/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/aiohttp/client.py:536: in _request\n    conn = await self._connector.connect(\n/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/aiohttp/connector.py:540: in connect\n    proto = await self._create_connection(req, traces, timeout)\n/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/aiohttp/connector.py:901: in _create_connection\n    _, proto = await self._create_direct_connection(req, traces, timeout)\n/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/aiohttp/connector.py:1209: in _create_direct_connection\n    raise last_exc\n/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/aiohttp/connector.py:1178: in _create_direct_connection\n    transp, proto = await self._wrap_create_connection(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <aiohttp.connector.TCPConnector object at 0x7fb32bd63250>\nreq = <aiohttp.client_reqrep.ClientRequest object at 0x7fb33035b9d0>\ntimeout = ClientTimeout(total=300, connect=None, sock_read=None, sock_connect=None)\nclient_error = <class 'aiohttp.client_exceptions.ClientConnectorError'>\nargs = (functools.partial(<class 'aiohttp.client_proto.ResponseHandler'>, loop=<_UnixSelectorEventLoop running=False closed=False debug=False>), '77.83.151.43', 443)\nkwargs = {'family': <AddressFamily.AF_INET: 2>, 'flags': <AddressInfo.AI_NUMERICHOST|AI_NUMERICSERV: 1028>, 'local_addr': None, 'proto': 6, ...}\n\n    async def _wrap_create_connection(\n        self,\n        *args: Any,\n        req: \"ClientRequest\",\n        timeout: \"ClientTimeout\",\n        client_error: Type[Exception] = ClientConnectorError,\n        **kwargs: Any,\n    ) -> Tuple[asyncio.Transport, ResponseHandler]:\n        try:\n            async with ceil_timeout(timeout.sock_connect):\n                return await self._loop.create_connection(*args, **kwargs)  # type: ignore[return-value]  # noqa\n        except cert_errors as exc:\n            raise ClientConnectorCertificateError(req.connection_key, exc) from exc\n        except ssl_errors as exc:\n            raise ClientConnectorSSLError(req.connection_key, exc) from exc\n        except OSError as exc:\n            if exc.errno is None and isinstance(exc, asyncio.TimeoutError):\n                raise\n>           raise client_error(req.connection_key, exc) from exc\nE           aiohttp.client_exceptions.ClientConnectorError: Cannot connect to host gosuslugi29.ru:443 ssl:False [Connection reset by peer]\n\n/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/aiohttp/connector.py:988: ClientConnectorError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"event_loop_policy","time":{"start":1733369788327,"stop":1733369788327,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"_session_event_loop","time":{"start":1733369788327,"stop":1733369788328,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"service_pages","time":{"start":1733370188841,"stop":1733370482006,"duration":293165},"status":"broken","statusMessage":"aiohttp.client_exceptions.ClientConnectorError: Cannot connect to host gosuslugi29.ru:443 ssl:False [Connection reset by peer]\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1139, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 897, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/pytest_asyncio/plugin.py\", line 366, in _async_fixture_wrapper\n    return event_loop.run_until_complete(setup())\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/asyncio/base_events.py\", line 654, in run_until_complete\n    return future.result()\n           ^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/pytest_asyncio/plugin.py\", line 363, in setup\n    res = await func(**_add_kwargs(func, kwargs, event_loop, request))\n          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/test-gosuslugi29/test-gosuslugi29/tests/test_service.py\", line 62, in service_pages\n    pages = await get_service_pages(session, request.config.services)\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/test-gosuslugi29/test-gosuslugi29/tests/test_service.py\", line 33, in get_service_pages\n    await responses\n  File \"/home/runner/work/test-gosuslugi29/test-gosuslugi29/utils/utils.py\", line 146, in fetch_url\n    async with session.request(**req, url=buildurl(**UrlPart)) as response:\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/aiohttp/client.py\", line 536, in _request\n    conn = await self._connector.connect(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/aiohttp/connector.py\", line 540, in connect\n    proto = await self._create_connection(req, traces, timeout)\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/aiohttp/connector.py\", line 901, in _create_connection\n    _, proto = await self._create_direct_connection(req, traces, timeout)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/aiohttp/connector.py\", line 1209, in _create_direct_connection\n    raise last_exc\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/aiohttp/connector.py\", line 1178, in _create_direct_connection\n    transp, proto = await self._wrap_create_connection(\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/aiohttp/connector.py\", line 988, in _wrap_create_connection\n    raise client_error(req.connection_key, exc) from exc\n","steps":[{"name":"Асинхронно сделать 100 запросов с 0 по 100 к страницам услуг","time":{"start":1733370188843,"stop":1733370481849,"duration":293006},"status":"broken","statusMessage":"aiohttp.client_exceptions.ClientConnectorError: Cannot connect to host gosuslugi29.ru:443 ssl:False [Connection reset by peer]\n","statusTrace":"  File \"/home/runner/work/test-gosuslugi29/test-gosuslugi29/tests/test_service.py\", line 33, in get_service_pages\n    await responses\n  File \"/home/runner/work/test-gosuslugi29/test-gosuslugi29/utils/utils.py\", line 146, in fetch_url\n    async with session.request(**req, url=buildurl(**UrlPart)) as response:\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/aiohttp/client.py\", line 536, in _request\n    conn = await self._connector.connect(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/aiohttp/connector.py\", line 540, in connect\n    proto = await self._create_connection(req, traces, timeout)\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/aiohttp/connector.py\", line 901, in _create_connection\n    _, proto = await self._create_direct_connection(req, traces, timeout)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/aiohttp/connector.py\", line 1209, in _create_direct_connection\n    raise last_exc\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/aiohttp/connector.py\", line 1178, in _create_direct_connection\n    transp, proto = await self._wrap_create_connection(\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/aiohttp/connector.py\", line 988, in _wrap_create_connection\n    raise client_error(req.connection_key, exc) from exc\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":1,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"check","time":{"start":1733370482180,"stop":1733370482180,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"afterStages":[{"name":"_session_event_loop::0","time":{"start":1733370535362,"stop":1733370535362,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"severity","value":"blocker"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_service"},{"name":"host","value":"fv-az1391-541"},{"name":"thread","value":"2072-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_service"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"blocker","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":3,"skipped":31,"passed":5,"unknown":0,"total":39},"items":[{"uid":"e9fd7779728377f4","reportUrl":"https://justfriendlyneighbor.github.io/test-gosuslugi29/90//#testresult/e9fd7779728377f4","status":"skipped","statusDetails":"Skipped: Completed succesfully, skipping from report","time":{"start":1733111072210,"stop":1733111072261,"duration":51}},{"uid":"a7eec47a1745403e","reportUrl":"https://justfriendlyneighbor.github.io/test-gosuslugi29/89//#testresult/a7eec47a1745403e","status":"skipped","statusDetails":"Skipped: Completed succesfully, skipping from report","time":{"start":1732765225451,"stop":1732765225500,"duration":49}},{"uid":"d24e47dd79def9f2","reportUrl":"https://justfriendlyneighbor.github.io/test-gosuslugi29/88//#testresult/d24e47dd79def9f2","status":"skipped","statusDetails":"Skipped: Completed succesfully, skipping from report","time":{"start":1732506099632,"stop":1732506099682,"duration":50}},{"uid":"f0ac9edff893476f","reportUrl":"https://justfriendlyneighbor.github.io/test-gosuslugi29/87//#testresult/f0ac9edff893476f","status":"skipped","statusDetails":"Skipped: Completed succesfully, skipping from report","time":{"start":1732169255784,"stop":1732169255834,"duration":50}},{"uid":"ceded409581942a9","reportUrl":"https://justfriendlyneighbor.github.io/test-gosuslugi29/86//#testresult/ceded409581942a9","status":"broken","statusDetails":"aiohttp.client_exceptions.ClientConnectorError: Cannot connect to host gosuslugi29.ru:443 ssl:False [Connection reset by peer]","time":{"start":1732160262337,"stop":1732160262337,"duration":0}},{"uid":"b518def08aab5d39","reportUrl":"https://justfriendlyneighbor.github.io/test-gosuslugi29/85//#testresult/b518def08aab5d39","status":"skipped","statusDetails":"Skipped: Completed succesfully, skipping from report","time":{"start":1731901204419,"stop":1731901204469,"duration":50}},{"uid":"5da09bdd428c14bf","reportUrl":"https://justfriendlyneighbor.github.io/test-gosuslugi29/84//#testresult/5da09bdd428c14bf","status":"skipped","statusDetails":"Skipped: Completed succesfully, skipping from report","time":{"start":1731555087332,"stop":1731555087381,"duration":49}},{"uid":"19c7d815292a7fd0","reportUrl":"https://justfriendlyneighbor.github.io/test-gosuslugi29/83//#testresult/19c7d815292a7fd0","status":"skipped","statusDetails":"Skipped: Completed succesfully, skipping from report","time":{"start":1731296385387,"stop":1731296385436,"duration":49}},{"uid":"b7584de32b389e16","reportUrl":"https://justfriendlyneighbor.github.io/test-gosuslugi29/82//#testresult/b7584de32b389e16","status":"skipped","statusDetails":"Skipped: Completed succesfully, skipping from report","time":{"start":1730950331133,"stop":1730950331184,"duration":51}},{"uid":"5aca55c9c859dfb8","reportUrl":"https://justfriendlyneighbor.github.io/test-gosuslugi29/81//#testresult/5aca55c9c859dfb8","status":"skipped","statusDetails":"Skipped: Completed succesfully, skipping from report","time":{"start":1730691347039,"stop":1730691347089,"duration":50}},{"uid":"ba3bdb0fe2e6d674","reportUrl":"https://justfriendlyneighbor.github.io/test-gosuslugi29/80//#testresult/ba3bdb0fe2e6d674","status":"skipped","statusDetails":"Skipped: Completed succesfully, skipping from report","time":{"start":1730362015431,"stop":1730362015479,"duration":48}},{"uid":"b9d0ecc43c6e637f","reportUrl":"https://justfriendlyneighbor.github.io/test-gosuslugi29/79//#testresult/b9d0ecc43c6e637f","status":"skipped","statusDetails":"Skipped: Completed succesfully, skipping from report","time":{"start":1730345699999,"stop":1730345700049,"duration":50}},{"uid":"3d144f01b9f47f1b","reportUrl":"https://justfriendlyneighbor.github.io/test-gosuslugi29/78//#testresult/3d144f01b9f47f1b","status":"skipped","statusDetails":"Skipped: Completed succesfully, skipping from report","time":{"start":1730300240625,"stop":1730300240674,"duration":49}},{"uid":"5978fc5f88bafefd","reportUrl":"https://justfriendlyneighbor.github.io/test-gosuslugi29/77//#testresult/5978fc5f88bafefd","status":"skipped","statusDetails":"Skipped: Completed succesfully, skipping from report","time":{"start":1730292450264,"stop":1730292450314,"duration":50}},{"uid":"a8867f2b9fadb5a5","reportUrl":"https://justfriendlyneighbor.github.io/test-gosuslugi29/76//#testresult/a8867f2b9fadb5a5","status":"skipped","statusDetails":"Skipped: Completed succesfully, skipping from report","time":{"start":1730097159696,"stop":1730097159745,"duration":49}},{"uid":"2efd426e414bacd2","reportUrl":"https://justfriendlyneighbor.github.io/test-gosuslugi29/75//#testresult/2efd426e414bacd2","status":"broken","statusDetails":"aiohttp.client_exceptions.ClientConnectorError: Cannot connect to host gosuslugi29.ru:443 ssl:False [Connect call failed ('77.83.151.43', 443)]","time":{"start":1730086546689,"stop":1730086546689,"duration":0}},{"uid":"d75abff5cc94a507","reportUrl":"https://justfriendlyneighbor.github.io/test-gosuslugi29/74//#testresult/d75abff5cc94a507","status":"skipped","statusDetails":"Skipped: Completed succesfully, skipping from report","time":{"start":1729839300789,"stop":1729839300838,"duration":49}},{"uid":"3be533692f1cc816","reportUrl":"https://justfriendlyneighbor.github.io/test-gosuslugi29/72//#testresult/3be533692f1cc816","status":"skipped","statusDetails":"Skipped: Completed succesfully, skipping from report","time":{"start":1729781339845,"stop":1729781339896,"duration":51}},{"uid":"28d5bb1295a652f7","reportUrl":"https://justfriendlyneighbor.github.io/test-gosuslugi29/71//#testresult/28d5bb1295a652f7","status":"skipped","statusDetails":"Skipped: Completed succesfully, skipping from report","time":{"start":1729779574306,"stop":1729779574358,"duration":52}},{"uid":"f787c1da2e8a246f","reportUrl":"https://justfriendlyneighbor.github.io/test-gosuslugi29/70//#testresult/f787c1da2e8a246f","status":"skipped","statusDetails":"Skipped: Completed succesfully, skipping from report","time":{"start":1729778374574,"stop":1729778374625,"duration":51}}]},"tags":[]},"source":"a729c66727d58961.json","parameterValues":[]}